README.md

```diff
@@ -7,25 +7,14 @@ Based on [php-github-api](https://github.com/m4tthumphrey/php-github-api) and co
 
 Installation
 ------------
-1. Install Composer
 
-    ```bash
-    $ curl -sS https://getcomposer.org/installer | php
-    $ sudo mv composer.phar /usr/local/bin/composer
-    ```
+Via [composer](https://getcomposer.org)
 
-2. Add the following to your require block in composer.json config.
-
-    > Note: be careful when using the `dev-master` tag as this may have unexpected results depending on your version of
-    Gitlab. See the Versioning section below for more information.
-
-    `php composer.phar require m4tthumphrey/php-gitlab-api:dev-master`
-
-3. Include Composer's autoloader:
+```bash
+composer require m4tthumphrey/php-gitlab-api php-http/guzzle6-adapter
```

Comment:
When using `php-http/guzzle6-adapter`, there is no need for `php-http/message`, right?